model/service_logs/v1/cluster_log_type.model

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+class ClusterLog {
+    Severity        String
+    ServiceName     String
```

Comment:
What is the meaning of this? Is this the cluster name? What is the difference between _service_ and _cluster_ in this context. Add documentation answering that.

Reply:
This should be the name of the service that created the log

We allow a service to create multiple logs per cluster(s) so it's supposed to be the service_name not specifically the cluster.

The service_name can be any text we like and it's useful to filter the logs by this name